react/src/views/BoxInfo.tsx

```diff
@@ -0,0 +1,48 @@
+import { useLazyQuery } from "@apollo/client";
+import React, { useEffect, useState } from "react";
+import { emptyBox } from "../utils/emptyBox";
+import { BOX_BY_QR } from "../utils/queries";
+
+function BoxInfo(props) {
+  const [box, setBox] = useState(emptyBox);
+
+  const [getBoxQuery] = useLazyQuery(BOX_BY_QR, {
+    onCompleted: (data) => {
+      var newBox = data.box;
+      setBox({
+        box_id: newBox.box_id,
+        product_id: newBox.product_id,
+        size_id: newBox.size_id,
+        items: newBox.items,
+        location_id: newBox.location_id,
+        comments: newBox.comments,
+        qr_id: newBox.qr_id,
+        box_state_id: newBox.box_state_id,
+      });
+    },
+    onError: (err) => {},
```

Comment:
Apollo error gets swallowed here (maybe ok for now - not sure whether the plan is to merge this PR already to master :) )